### PR TITLE
Added fingerprints for IKEA bulbs: GU10 and E12

### PR DESCRIPTION
**Pull request by MMateusiakS**
The IKEA device "TRADFRI bulb GU10 W op/ch 400lm" monitors itself as model "TRADFRI bulb E17 W op/ch 400lm". For current situation, for every bulb that claims to be model "E17" we will display in the Smarthings/ Samsung Connect App the model "GU10".

### CODE REVIEW

**Review by greens — DISMISSED** [review]
Can we get a version that's not confused by whitespace changes?

File: devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy [hunk 1]
```diff
@@ -38,6 +38,9 @@ metadata {
...
         fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 1000", outClusters: "0019", manufacturer: "Philips", model: "LWB014", deviceJoinName: "Philips Hue White"
         fingerprint inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "TRADFRI bulb E26 opal 1000lm", deviceJoinName: "TRADFRI bulb E26 opal 1000lm"
+		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "TRADFRI bulb E12 W op/ch 400lm", deviceJoinName: "TRADFRI bulb E12 W op/ch 400lm"
+		//IKEA bulb monitors model as "TRADFRI bulb E17", but in reality the model is "TRADFRI bulb GU10"
+		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "TRADFRI bulb E17 W op/ch 400lm", deviceJoinName: "TRADFRI bulb GU10 W op/ch 400lm"
```
Comment (MMateusiakS) [inline devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy]: Hi, @greens. I added some changes as you asked. But I am concerned about how it looks like.  Are you Ok with this?

**Review by greens — DISMISSED** [review]
It's okay. I double-checked that the spacing matched in my IDE. This is just how github formats tabs v. spaces.

File: devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy [hunk 2]
```diff
@@ -39,8 +39,9 @@ metadata {
...
 		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "TRADFRI bulb E12 W op/ch 400lm", deviceJoinName: "TRADFRI bulb E12 W op/ch 400lm"
-		//IKEA bulb monitors model as "TRADFRI bulb E17", but in reality the model is "TRADFRI bulb GU10"
-		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "TRADFRI bulb E17 W op/ch 400lm", deviceJoinName: "TRADFRI bulb GU10 W op/ch 400lm"
+		//IKEA bulb GU10 monitors model as TRADFRI bulb "E17" or "GU10"
+		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "TRADFRI bulb E17 W op/ch 400lm", deviceJoinName: "TRADFRI bulb 400lm"
+		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "TRADFRI bulb GU10 W 400lm", deviceJoinName: "TRADFRI bulb 400lm"
```
Comment (MMateusiakS) [inline devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy]: @greens We have issue about IKEA light bulbs. We have 3 bulbs of IKEA TRADRI GU10. Two of them report that their model is "E17" and one report that is "GU10". So to solve current issue I added one more fingerprint and change the device join name to not cointain information about bulb screw in this particular model.

**Review by greens — APPROVED** [review]
This seems like the right solution for the problem.

**Comment by MMateusiakS** [comment]
Mariusz Ablewicz has already commited the changes.

**Pull request closed without merging**